### ISSUE TITLE
NTR: Endoscopy: pharyngeal mass (GI, feeding, vomiting & dehydration)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7883]
New term request Endoscopy: pharyngeal mass (elthat_150214030952): Pharynx  UMLS: 0747551

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000220
- label: Pharyngeal mass
- parents: [Abnormal pharynx morphology (HP:0033151)](https://hpo.jax.org/app/browse/term/HP:0033151)
- pmid: 
- def: An abnormal enlargement or swelling localized to the pharynx (i.e., the throat, the space that connects the nose and mouth to the larynx and esophagus). The word mass is usually used at an early stage of the diagnostic workup before the precise nature of the swelling has been determined.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7883
- synonyms:
